src/bin/ch-remote.rs

```diff
@@ -930,14 +939,15 @@ fn coredump_config(destination_url: &str) -> String {
     serde_json::to_string(&coredump_config).unwrap()
 }
 
-fn receive_migration_data(url: &str) -> String {
+fn receive_migration_data(url: &str, tls_dir: Option<&PathBuf>) -> String {
```

Comment:
```suggestion
fn receive_migration_data(url: &str, tls_dir: Option<&Path>) -> String {
```

`&Path` to `PathBuf` behaves similar like `&str` to `String`. You barely ever use `&String` and so you also barely ever use `&PathBuf`

**But I messed things up.** We take ownership anyway. sorry. So we can consume the owned type.

```suggestion
fn receive_migration_data(url: &str, tls_dir: Option<PathBuf>) -> String {
```

Reply:
Fixed.